src/main/java/taxi/controllers/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package taxi.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.security.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("taxi");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("taxi");
```
it's not a constant, cause the object is not deeply immutable